common/source/docs/common-smartap-pdb.rst

```diff
@@ -0,0 +1,73 @@
+.. _common-smartap-pdb:
+
+===========================================
+Sky-Drones SmartAP Power Distribution Board
+===========================================
+
+Overview
+========
+
+`SmartAP PDB <https://www.mauch-electronic.com/#webs-bin-563c137b7add4cb229080f55>`__ (Power Distribution Board) is a board which allows transferring the power from 
+the battery to ESCs / Motors and generate power supply for the flight controller and 
+other peripherals with different voltage levels. Also, PDB provides the functionality for battery 
+voltage / current measurements. SmartAP PDB makes high-power lines connections easier and much more reliable.
+
+
+.. image:: ../../../images/smartap-pdb/smartap-pdb-top-side.jpg
+    :target: ../_images/smartap-pdb/smartap-pdb-top-side.jpg
+
+Specifications
+==============
+
+- Size: 65x65 mm, four 3mm mounting holes
+- Input voltage up to 60 Volts (14S)
+- Capability to handle extremenly high currents (peak current up to 400A)
+- Power input from the main battery, possibility to connect up to 4 independent batteries
+- 12 pairs of pads (6 on top, 6 on bottom) for powering up to 12 motors (all possible airframe configurations supported)
+- Integrated voltage and current sensors
+- Integrated DC-DC converter from 10-60 V input (up to 14S battery) to 5V output to power peripherals
+- Integrated DC-DC converter from 10-60 V input (up to 14S battery) to 12V output to power peripherals
+- 5V and 12V power output terminals (standard 2.54mm/0.1" connectors)
+- Integrated loud electromagnetic sounder (buzzer)
+- Power output for the flight controller (both 5V and battery VIN)
+- Fully compatible with SmartAP, APM and PX4 autopilots
+
+Size and Weight
+===============
+
+- Length: 65mm
+- Width: 65mm
+- Height: 14mm
+- Weight: 16g
+
+
+Buy
+===
+
+Sky-Drones store - purchase `here <https://sky-drones.com/parts/smartap-pdb.html>`__
+
+Pinouts
+=======
+
+SmartAP Power Distribution Board pinout diagram is shown below. Big pads in the rear side are intended for the main battery connection. Up to four independent batteries can be connected using the thick wires (e.g. 8-10 AWG) to be able to handle high current loads and even more using thinner wires. Both top and bottom, left and right sides have pads for ESCs power supply connection. Therefore, up to 12 ESCs can be connected.
+
+.. image:: ../../../images/smartap-pdb/smartap-pdb-pinout.png
+    :target: ../_images/smartap-pdb/smartap-pdb-pinout.png
+
+
+Voltage and current sensors
+===========================
+
+SmartAP PDB has integrated voltage and current sensors. Current sensor is located on the bottom side of the PDB. For correct scale / offset configuration, please, refer to Configuration section.
+
+.. image:: ../../../images/smartap-pdb/smartap-pdb-current-sensor.png
+    :target: ../_images/smartap-pdb/smartap-pdb-current-sensor.png
+
+
+Configuration
+=============
+
+- Voltage divider: 8.33
```

Comment:
These should refer to the ArduPilot  parameters...ie :ref:`BAT_VOLT_MULT<BAT_VOLT_MULT>` =  8.33, etc.
and the value of 8.33 is a bit strange, since you state that it can support 60V inputs....which would overdrive a 3.3V analog ADC input with 7.2V when the input is 60V....

Reply:
due to Markdown differences, the param name being surrounded by backticks shows here as highlighting....look at a wiki page source to see the formatting for param names...

Reply:
Thanks for noticing, my mistake, this was calculated using the different approach. Now calculated correctly and fixed. 

Parameters styling applied.